de-lib/src/main/java/org/iplantc/de/admin/desktop/client/toolAdmin/view/dialogs/ToolAdminDetailsDialog.java

```diff
@@ -94,24 +86,9 @@ public void show(final Tool tool, Mode mode) {
 
     @Override
     public void show() {
-        final ToolContainer toolContainer = factory.getContainer().as();
-        toolContainer.setDeviceList(Lists.<ToolDevice>newArrayList());
-        toolContainer.setContainerVolumes(Lists.<ToolVolume>newArrayList());
-        toolContainer.setContainerVolumesFrom(Lists.<ToolVolumesFrom>newArrayList());
-        ToolImage image = factory.getImage().as();
-        image.setName(appearance.defaultImgPrefix());
-        toolContainer.setImage(image);
-
-        final ToolImplementation toolImplementation = factory.getImplementation().as();
-        final ToolTestData toolTestData = factory.getTest().as();
-        toolTestData.setInputFiles(Lists.<String>newArrayList());
-        toolTestData.setOutputFiles(Lists.<String>newArrayList());
-        toolImplementation.setTest(toolTestData);
```

Comment:
Oh I didn't realize these were removed.  You might wanna test.  I vaguely remember I had to add all those empty lists to make sure the JSON structure was in tact.  It was something weird like if you didn't create the nested structure, then when you went to save something within ToolContainer or ToolTestData for example, those lists wouldn't actually save because it needed a list to already be there?  Something weird like that, I honestly don't remember the details after all this time though.  It could be fixed after the updates/upgrades we've done though.